site/_docs/templates.md

```diff
@@ -23,6 +23,26 @@ common tasks easier.
   <tbody>
     <tr>
       <td>
+        <p class="name"><strong>Array Filters</strong></p>
+        <p>Push, pop, shift, and unshift elements from an Array.</p>
+      </td>
+      <td class="align-center">
+        <p>
+         <code class="filter">{% raw %}{{ page.tags | push: 'Spokane' }}{% endraw %}</code>
+         <code class="filter">{% raw %}{{ page.tags | pop }}{% endraw %}</code>
+         <code class="filter">{% raw %}{{ page.tags | shift }}{% endraw %}</code>
+         <code class="filter">{% raw %}{{ page.tags | unshift }}{% endraw %}</code>
```

Comment:
`unshift` adds an item to the beginning of an array. In this case, it should have an argument:

```liquid
{{ page.tags | unshift: "Olympia" }}
```